dir2module/dir2module.py

```diff
@@ -125,11 +125,11 @@ def has_modularity_label(self):
     def _get_header(self):
         """
         Examine a RPM package file and return its header
-        See https://docs.fedoraproject.org/en-US/Fedora_Draft_Documentation/0.1/html/RPM_Guide/ch16s04.html
+        See docs.fedoraproject.org/en-US/Fedora_Draft_Documentation/0.1/html/RPM_Guide/ch16s04.html
```

Comment:
A smart workaround, but overall I don't know how to deal with long URLs in docstrings. Do you?